src/StoreApi/Routes/Checkout.php

```diff
@@ -330,53 +333,44 @@ protected function is_valid_draft_order( $order_object ) {
 		return false;
 	}
 
-	/**
-	 * Get an order object, either using a current draft order, or returning a new one.
-	 *
-	 * @param integer $order_id Draft order ID.
-	 * @return \WC_Order|boolean Either the draft order, or false if one has not yet been created.
-	 */
-	protected function get_draft_order_object( $order_id ) {
-		// If order ID doesn't exist or it doesn't match a draft order, create
-		// a new draft order. That might happen when trying to checkout directly
-		// from the Cart block with an express payment method and the draft
-		// order hasn't been created yet.
-		$draft_order_object = $order_id ? wc_get_order( $order_id ) : false;
-
-		return $this->is_valid_draft_order( $draft_order_object ) ? $draft_order_object : $this->create_or_update_draft_order();
-	}
-
 	/**
 	 * Create or update a draft order based on the cart.
 	 *
 	 * @throws RouteException On error.
-	 *
-	 * @return \WC_Order Order object.
 	 */
-	protected function create_or_update_draft_order() {
+	private function create_or_update_draft_order() {
 		$cart_controller  = new CartController();
 		$order_controller = new OrderController();
 		$reserve_stock    = \class_exists( '\Automattic\WooCommerce\Checkout\Helpers\ReserveStock' ) ? new \Automattic\WooCommerce\Checkout\Helpers\ReserveStock() : new ReserveStock();
-		$order_object     = $this->get_draft_order_id() ? wc_get_order( $this->get_draft_order_id() ) : null;
 		$created          = false;
```

Comment:
I know this variable wasn't introduced in this pr but I noticed in reviewing, what purpose does `$created` have in this method. It doesn't seem to be implemented anywhere (other than switched to `true` when creating the order)?